todo/management/commands/import_csv.py

```diff
@@ -0,0 +1,56 @@
+import sys
+from typing import Any
+from pathlib import Path
+
+from django.core.management.base import BaseCommand, CommandParser
+
+from todo.operations.csv_importer import CSVImporter
+
+
+class Command(BaseCommand):
+    help = """Import specifically formatted CSV file containing incoming tasks to be loaded.
+    For specfic format of inbound CSV, see data/import_example.csv.
+    For documentation on field formats and required fields, see README.md.
+    """
+
+    def add_arguments(self, parser: CommandParser) -> None:
+
+        parser.add_argument(
+            "-f", "--file", dest="file", default=None, help="File to to inbound CSV file."
+        )
+
+    def handle(self, *args: Any, **options: Any) -> None:
+        # Need a file to proceed
+        if not options.get("file"):
+            print("Sorry, we need a file name to work from.")
+            sys.exit(1)
+        else:
+            filepath = str(options.get("file"))
+            if not Path(filepath).exists():
+                print(f"Sorry, couldn't find file: {filepath}")
+                sys.exit(1)
+
+        with open(filepath, mode="r", encoding="utf-8-sig") as fileobj:
```

Comment:
why `utf-8-sig` and not just `utf-8` ?

Reply:
Ah, this one is interesting. From a problem recently solved at work. When someone opens a CSV in Excel (Windows or Mac) and saves to CSV, it inserts a hidden byte-order-mark which throws off the column count/tracking. `sig` means "ignore BOM" and makes it work whether someone works in Excel or a more respectful app.

Reply:
FWIW this doesn't happen if you just open a CSV in Excel and save - it has to exist as an .xlsx and then saved to CSV... which is a common workflow.